nucypher/blockchain/eth/actors.py

```diff
@@ -501,7 +501,7 @@ def save_deployment_receipts(self, receipts: dict, filename_prefix: str = 'deplo
             file.write(data)
         return filepath
 
-    def set_min_fee_rate_range(self,
+    def set_fee_rate_range(self,
                                minimum: int,
```

Comment:
lines are not aligned after renaming

Reply:
✔️

Reply:
also there are few more places where is the same